model/src/test/java/com/javahelp/model/user/UserPasswordTest.java

```diff
@@ -0,0 +1,150 @@
+package com.javahelp.model.user;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+import java.util.Base64;
+import java.util.Random;
+
+/**
+ * A testing file for UserPassword entity
+ */
+public class UserPasswordTest {
+
+    /**
+     * The current salt for the first user
+     */
+    private byte[] saltOne;
```

Comment:
Just to emphasize, this is the correct format for Javadocs, but in a situation like this (on a private field, especially one in a test class) they are not necessarily required. This is the right format though.

This isn't a request for changes, just a note for the future.